Subject: [PATCH] systemui: fix TunerActivity crash

Status bar icon settings currently crashes on
screen rotate or pressing recents with:
java.lang.IllegalStateException: Failure saving state:
SubSettingsFragment{b614cb2 #1 id=0x7f120150} has target
not in fragment manager: TunerFragment{1781403}

diff --git a/packages/SystemUI/src/com/android/systemui/tuner/TunerActivity.java b/packages/SystemUI/src/com/android/systemui/tuner/TunerActivity.java
index cd6474e..084daf7 100644
--- a/packages/SystemUI/src/com/android/systemui/tuner/TunerActivity.java
+++ b/packages/SystemUI/src/com/android/systemui/tuner/TunerActivity.java
@@ -56,6 +56,11 @@
     }
 
     @Override
+    public void onSaveInstanceState(final Bundle outState) {
+        // save nothing
+    }
+
+    @Override
     public void onBackPressed() {
         if (!getFragmentManager().popBackStackImmediate()) {
             super.onBackPressed();
